src/app/events/page.tsx

```diff
@@ -76,6 +84,12 @@ export const metadata = createMetadata({
   overrideDefaultTitle: true,
 })
 
+const locationOptions = getCMSOptionsWithDefault({
```

Comment:
Should we abstract some of this config?

```
export const EVENT_CMS_OPTIONS = {
  location: {
    collectionName: 'event_entries',
    fieldName: 'location.region',
    defaultOption: DEFAULT_LOCATION_FILTER_OPTION,
  },
  category: {
    collectionName: 'event_entries',
    fieldName: 'category',
  }
} as const
```

Reply:
I went with `EVENT_FILTER_OPTION_SETTINGS`, feel free to rename it if you have a better suggestion

Reply:
I renamed it to `FILTERS_CONFIG` - Shorter and it's scoped under Events either way. 🙏🏼